db/db_tunables.h

```diff
@@ -1659,6 +1659,11 @@ REGISTER_TUNABLE("dohsql_max_queued_kb_highwm",
                  TUNABLE_INTEGER, &gbl_dohsql_max_queued_kb_highwm, 0, NULL,
                  NULL, NULL, NULL);
 
+REGISTER_TUNABLE("dohast_disable_joins",
```

Comment:
Call it "dohast_parallel_joins" instead and disable it by default? "disable something to enable" sounds a bit counter-intuitive.

Reply:
disable set to 1, disables, not enables.

Reply:
Sorry, may be I was not clear enough. I think, having `disable` or `enable` in the tunable name is a bit confusing. That's what the tunable values are for : `OFF/ON` `0/1`.

In order to enable this feature, I'd prefer:
`
dohast_parallel_joins ON
`

over,

`
dohast_disable_joins OFF
`

Said that, this suggestion is purely optional, If I you aren't convinced, feel free to ignore it. :)